facilitator/src/bin/facilitator.rs

```diff
@@ -646,3 +710,16 @@ fn public_key_from_arg(arg: &str, matches: &ArgMatches) -> UnparsedPublicKey<Vec
         Err(_) => UnparsedPublicKey::new(&ECDSA_P256_SHA256_FIXED, key_bytes),
     }
 }
+
+fn transport_for_output_path(arg: &str, matches: &ArgMatches) -> Result<Box<dyn Transport>> {
+    let path = parse_path(matches.value_of(arg).unwrap())?;
```

Comment:
I think just to hide an `unwrap` call behind this function, and put less noise at the callsite. That said a subsequent PR does end up pulling more than one value from `matches` here (to configure whether the `S3Transport` should fetch an auth token from instance metadata or use AWS creds from the environment) so this will end up being useful soon.